contracts/PathsController.sol

```diff
@@ -0,0 +1,53 @@
+pragma solidity ^0.4.8;
+
+import "./PathsDB.sol";
+
+contract PathsController {
+    address public owner;
+    mapping (uint => address) public paths;
+
+    function PathsController() {
+        owner = msg.sender;
+    }
+
+    function createPath(uint _lotId, address _checkpoint) returns (bool) {
+        PathsDB pathsDB = new PathsDB();
+        pathsDB.addCheckpoint(_checkpoint);
+        paths[_lotId] = address(pathsDB);
+
+        return true;
+    }
+
+    function addCheckpoint(uint _lotId, address _checkpoint) returns (bool) {
+        bool result = PathsDB(paths[_lotId]).addCheckpoint(_checkpoint);
```

Comment:
@DavidKnott Just for curiosity/learning - how is this line different from lines 14 and 15 above?